shotover-proxy/benches/windsock/profilers/sar.rs

```diff
@@ -121,7 +122,7 @@ pub fn parse_sar(rx: &mut UnboundedReceiver<String>) -> ParsedSar {
         };
     };
     let started_at =
-        OffsetDateTime::from_unix_timestamp_nanos(started_at.parse().unwrap()).unwrap();
+        OffsetDateTime::from_unix_timestamp_nanos(started_at.unwrap().parse().unwrap()).unwrap();
```

Comment:
unwrap() the unwrap() the unwrap() :)

Reply:
not sure if you were just amused at all the unwraps but this is test code so unwrap is the most useful tool for the job here since it immediately terminates when something goes wrong and gives us a nice stack trace to investigate.

Sometimes we can give better diagnostics to the developer by handling errors so that we can, for example, report multiple things that went wrong instead of just one.
but I dont see that as being the case here.